tests/test_core.py

```diff
@@ -67,6 +67,25 @@ def test_commit_nonblocking(self):
             d['key'] = 'value'
             d.commit(blocking=False)
 
+    def test_special_keys(self):
+        """integer, float and/or tuple keys"""
+        db = SqliteDict()
+        db['1'] = 1
+        db[1] = 'ONE'
+        db[('a',1)] = 'testtuple'
+        db[frozenset([1,2,'2'])] = 'testfrozenset'
```

Comment:
this should have assertions about the underlying generated key names, possibly as separate test method, but also here so it is possible to understand the implementation by reading the tests